template/build/webpack.prod.conf.js

```diff
@@ -70,6 +71,26 @@ module.exports = merge(baseWebpackConfig, {
     new webpack.optimize.CommonsChunkPlugin({
       name: 'manifest',
       chunks: ['vendor']
+    }),
+    // auto generate service-worker for caching static files
+    new SWPrecachePlugin({
+      cacheId: 'quasar-app',
+      filename: 'service-worker-cache.js',
+      minify: true,
+
+      staticFileGlobs: [
+        'js/**.js',
+        '/**.css'
+      ],
+
+      runtimeCaching: [{
+        urlPattern: /\/.*/,
+        handler: 'networkFirst'
```

Comment:
I think you're right, cache-first approach seems to be very nice on mobile. The problem is that this configuration has to be changed based on projects. There is no configuration that fit well on every project, some needs network-first, some need fastest, depending always on what resources are we caching. Thanks though for the tip. Let's talk again after v0.14